kong/clustering/services/sync/hooks.lua

```diff
@@ -161,8 +166,38 @@ function _M:register_dao_hooks()
 
     ngx_log(ngx_DEBUG, "[kong.sync.v2] new delta due to deleting ", name)
 
-    -- set lmdb value to ngx_null then return row
-    return self:entity_delta_writer(entity, name, options, ws_id, true)
+    -- set lmdb value to ngx_null then return entity
+
+    local deltas = { gen_delta(entity, name, options, ws_id, true) }
```

Comment:
```suggestion
    local deltas = { (gen_delta(entity, name, options, ws_id, true)) }
```

Just worrying if the function returns more than one result and it will be a hard to debug issue

Reply:
Currently `gen_delta()` returns only one table, there is no other exit point.

Reply:
changed, thanks.